acceptance/testdata/mock_buildpacks/v1/builder.toml

```diff
@@ -1,3 +1,5 @@
+schema = "2"
```

Comment:
i noticed this is in a dir `v1`. is that related to schema version (and is it correct)?

Reply:
This is the schema of the builder.toml, which is 2 in both cases (v1 dir and v2 dir) because it's always using the latest pack.

Reply:
We'll be removing references to `schema` in this PR for now, until we can discuss further